cypress/integration/channel/system_message_spec.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+function getLines(e) {
+    const $cont = Cypress.$(e);
+    const textArr = $cont.text().split(' ');
+
+    for (let i = 0; i < textArr.length; i++) {
+        textArr[i] = '<span>' + textArr[i] + ' </span>';
+    }
+
+    $cont.html(textArr.join(''));
+
+    const $wordSpans = $cont.find('span');
+    const lineArray = [];
+    var lineIndex = 0;
+    var lineStart = true;
+
+    $wordSpans.each(function handleWord(idx) {
+        const top = Cypress.$(this).position().top;
+
+        if (lineStart) {
+            lineArray[lineIndex] = [idx];
+            lineStart = false;
+        } else {
+            var $next = Cypress.$(this).next();
+
+            if ($next.length) {
+                if ($next.position().top > top) {
+                    lineArray[lineIndex].push(idx);
+                    lineIndex++;
+                    lineStart = true;
+                }
+            } else {
+                lineArray[lineIndex].push(idx);
+            }
+        }
+    });
+    return lineArray.length;
+}
+
+describe('System Message', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('MM-14636 - Validate that system message is wrapping properly', () => {
+        // 2. Open channel header textbox
+        cy.get('#channelHeaderDropdownButton').
+            should('be.visible').
+            click();
+        cy.get('#channelHeaderDropdownMenu').
+            should('be.visible').
+            find('#channelEditHeader').
+            click();
+
+        // 3. Enter short description
+        cy.get('#edit_textbox').
+            clear().
+            type('> newheader').
+            type('{enter}').
+            wait(500);
+
+        cy.getLastPost().
+            should('contain', 'System').
+            and('contain', 'user-1 updated the channel');
```

Comment:
I suggests to add `@` to exactly match the message.  
```suggestion
            and('contain', '@user-1 updated the channel');
```

Also, this test will fail in case the teammate name display setting (at "Account Settings > Display > Teammate Name Display") is not set to "Show username".
I recommend to set it to username in "before" hook with the  new helper command like `cy.updateTeammateNameDisplayPreference()` that directly updates the setting via API (see [this command](https://github.com/mattermost/mattermost-webapp/blob/c84a8fd737c05316f98da3cef89176a34f56a25a/cypress/support/commands.js#L461) for reference).

Reply:
Oh I missed to mention that the requirement is "System messages should display correctly" which means it should not only match the "@user-1 updated the channel" but the entire system message.  It should include checking of "@user-1 updated the channel from: {old channel header} to: {new channel header}".